pkg/urbit/vere/saga.c

```diff
@@ -375,7 +375,7 @@ u3_saga_new(const c3_path* const pax_u, const u3_meta* const met_u)
 }
 
 u3_saga*
-u3_saga_open(const c3_path* const pax_u, u3_meta* const met_u)
+_saga_open(const c3_path* const pax_u, u3_meta* const met_u, c3_i* san_i)
```

Comment:
`static` functions should go in the "Static functions" section of the file, which is alphabetized by `static` function name and which begins with declarations of all of the `static` functions (to enable alphabetization of the function definitions).